gitlab/v4/objects.py

```diff
@@ -1764,6 +1768,12 @@ def create(self, data, **kwargs):
         return CreateMixin.create(self, data, path=path, **kwargs)
 
 
+class PipelineJobManager(ListMixin, RESTManager):
+    _path = '/projects/%(project_id)s/pipelines/%(pipeline_id)s/jobs'
+    _obj_cls = ProjectJob
```

Comment:
Although this seems to be a good idea to use a ProjectJob, things get a bit more complicated than expected when using the CLI. To avoid bad surprises I try to stick to the upstream API logic: we are requesting a different URL so we get a different object class.

Could you create and use a `PipelineJob` object class instead?